util/executil/executil.go

```diff
@@ -25,14 +26,17 @@ const (
 
 // HandleExecError wraps an error returned by a function on exec.Cmd, adding
 // stderr to the message if the error is an exec.ExitError.
+// Note: exec.ExitError instances are logged without the stack trace and
+// returned as a cmdutils.SilentError.
 func HandleExecError(cmd *exec.Cmd, err error) error {
 	if exitErr, ok := err.(*exec.ExitError); ok {
 		stderr := string(exitErr.Stderr)
 		if stderr != "" && !strings.HasSuffix(stderr, "\n") {
 			stderr += "\n"
 		}
 		err = fmt.Errorf("%s%s: %w", stderr, cmd.Args[0], err)
-		return errors.WithStack(err)
+		log.Error(err)
```

Comment:
Our `log.Error` function takes the error as the first argument and the message it should print as the second. If no message is provided, it prints an empty string (we should probably change that).
```suggestion
		log.Error(err, err.Error())
```